libs/ui-lib/lib/ocm/services/ClusterDetailsService.ts

```diff
@@ -45,7 +45,10 @@ const ClusterDetailsService = {
       },
     };
 
-    if (params.cpuArchitecture === CpuArchitecture.ARM || params.platform?.type === 'oci') {
+    if (
+      !values.isCMNSupported &&
+      (params.cpuArchitecture === CpuArchitecture.ARM || params.platform?.type === 'oci')
```

Comment:
```suggestion
```
This needs to be gone. Whether UMN is supported or not should be driven by the support-levels API